controller/src/main/java/io/pravega/controller/store/stream/Scope.java

```diff
@@ -39,6 +43,35 @@
      */
     CompletableFuture<Void> deleteScope();
 
+    /**
+     * Api to add stream under a scope for listing streams. 
+     * 
+     * @param name name of stream
+     * @param creationTime  creation time of stream.
+     * @return Future, which upon completion will indicate that the stream has been added to the scope. 
+     */
+    CompletableFuture<Void> addStreamToScope(final String name, final long creationTime);
```

Comment:
How are these related to this pull request?

Reply:
this is coming from issue #3278